script/publish_views

```diff
@@ -58,7 +60,15 @@ def process_file(client, args, filepath):
         if args.dry_run:
             print(f"Validated definition of {target_view} in {filepath}")
         else:
-            query_job.result()
+            try:
+                query_job.result()
+            except BadRequest as e:
+                if ("Invalid snapshot time" in e.message):
+                    # This occasionally happens due to dependent views being
+                    # published concurrently; we wait briefly and give it one
+                    # extra try in this situation.
+                    time.sleep(1)
+                    client.query(sql, job_config).result()
```

Comment:
or else we'd want to raise the same exception again to make it fail (in cases where it's a `BadRequest` but not due to invalid snapshot times)